Frameworks/UIKit/UIButton.mm

```diff
@@ -784,17 +872,18 @@ - (UIEdgeInsets)contentEdgeInsets {
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
 */
 - (void)setShowsTouchWhenHighlighted:(BOOL)showsTouch {
     UNIMPLEMENTED();
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
+ @Notes Xaml doesn't have built in support for showing 'glowing' touch feedback.
 */
 - (BOOL)showsTouchWhenHighlighted {
-    UNIMPLEMENTED();
+    UNIMPLEMENTED_WITH_MSG("Xaml doesn't have built in support for showing 'glowing' touch feedback.");
```

Comment:
looks like this is the only one i missed

---
In reply to: [99922832](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r99922832) [](ancestors = 99922832,99914391)